src/main/java/com/github/manolo8/darkbot/backpage/NovaManager.java

```diff
@@ -84,13 +85,14 @@ private void updateRosterList() throws IOException {
         if (jsonObj.get("result").getAsString().equalsIgnoreCase("OK")) {
             String resp = jsonObj.getAsJsonArray("rosterList").toString();
             Agent[] rosterList = gson.fromJson(resp, Agent[].class);
-
             for (Agent agent : rosterList) {
                 this.data.addAgent(agent);
             }
+            return false;
```

Comment:
shouldn't it be true here for "properly updated" ? what does true/false mean in this method?

Reply:
haha, i think too fast!!!